android/library/src/main/java/io/logto/android/Logto.kt

```diff
@@ -37,15 +38,19 @@ object Logto {
         onComplete: (error: Error?, credential: Credential?) -> Unit
     ) {
         checkInitState()
-        BrowserLoginFlow.init(
-            logtoConfig,
-        ) { error, credential ->
-            if (error == null && credential != null) {
-                credentialCache = credential
-                credentialStorage?.saveCredential(credential)
+        AuthManager.start(
```

Comment:
thought `AuthManager` will be a high level abstract instead of an additional layer.. let's go through the docs first

Reply:
TBD: [LOG-149](https://linear.app/silverhand/issue/LOG-149/research-best-practice-in-browser-flow)